### PR TITLE
Adversarial VAE

### PR DESCRIPTION
**Pull request by shujonnaha**

### CODE REVIEW

**Comment by piiswrong** [comment]
Thanks for the contribution.

Please remove __init__ and add ad README.md to explain usage and reference

**Comment by piiswrong** [comment]
Please add a license header to all your .py files like the other examples and we can merge.

Thanks

**Comment by piiswrong** [comment]
@shujonnaha Could you add license header?

**Comment by shujonnaha** [comment]
Hi
Sure, I will do it by today.


Shujon Naha
Ph.D. Student
Computer Science
Indiana University

File: example/mxnet_adversarial_vae/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+VAE-GAN in MxNet
```
Comment (tornadomeet) [inline example/mxnet_adversarial_vae/README.md]: MxNet --> MXNet

**Comment by piiswrong** [comment]
Thanks

**Pull request merged**